elixir/apps/web/lib/web/router.ex

```diff
@@ -66,12 +66,6 @@ defmodule Web.Router do
     live "/", SignUp
   end
 
-  scope "/try", Web do
-    pipe_through :public
-
-    live "/", TempAccounts.Index
-  end
```

Comment:
Was wondering if we should keep this path and just have it redirect to `/sign_up`?  Just thinking if anyone comes across the Launch HN then at least that link will still work technically.  We could even have a small delay saying "Temporary accounts have been removed, but you can get started with a free account here"

Reply:
Yeah good call, I'll just redirect to `/sign_up`.